OpenSim/Actuators/VandenBogert2011Muscle.h

```diff
@@ -0,0 +1,568 @@
+        static equilibrium version of computeInitialFiberEquilibrium(). By
+        setting velocities to zero, we obtain a reasonable and robust solution
+        that provides a rough solution for fiber length.
+                @param[in,out] s The state of the system. */
+        void computeFiberEquilibriumAtZeroVelocity(SimTK::State& s) const
+                override;
+
+
+//=============================================================================
+// COMPUTATION
+//=============================================================================
+
+    // TODO: These should move to protected at some point
+
+        // Methods to calculate muscle residual:
+
+
+        /** Calculates the muscle residuals.
+                @param muscleLength length of muscle (N)
+                @param projFibLength length of the muscle fiber projected inline
+                        with tendon (m)
+                @param activ muscle activation
+                @param projFiberVelocity velocity of the muscle fiber projected
+                        inline with tendon (m/s)
+                @param activ_dot time derivative of the muslce activation
+                @param u muscle control (excitation)
+                @param returnJacobians if=1, calculate and return the
+                        Jacobains. default is =0, no Jacobians not calculated
+                @returns TBD for now see ImplicitResidual structure*/
+        ImplicitResidual calcImplicitResidual(double muslceLength,
+                                              double projFiberLength,
+                                              double activ,
+                                              double projFiberVelocity,
+                                              double activ_dot,
+                                              double u,
+                                              int returnJacobians) const;
+        /** Calculates the muscle residuals. (State equation form for vector
+        manipulation)
+                @param y vector of:
+                        \li 0: projected fiber length
+                        \li 1: muscle activation
+                @param ydot_guess Vector of the guess for the derivative of y
+                @param muscleLength length of muscle (N)
+                @param u muscle control (excitation)
+                @param returnJacobians if=1, calculate and return the
+                        Jacobains. default is =0, no Jacobians not calculated
+                @returns TBD for now see ImplicitResidual structure*/
+        ImplicitResidual calcImplicitResidual(SimTK::Vec2 y,
+                                              SimTK::Vec2 ydot_guess,
+                                              double muscleLength,
+                                              double u,
+                                              int returnJacobians) const;
+
+        /** Calculates the muscle residuals. (method for getting from
+        state)
```

Comment:
If these three signatures remain, it would be good to use a doxygen group to group them together and clarify that they all do the exact same thing but just with different signatures.